googlecloud-plugins/docs/BigQuery-batchsource.md

```diff
@@ -0,0 +1,69 @@
+# Google BigQuery Batch Source
+
+
+Description
+-----------
+Batch source to use Google BigQuery as a Source.
+[link to BigQuery Docs](https://cloud.google.com/bigquery/docs/)
+
+Use Case
+--------
+This source is used whenever you need to read from a Google BigQuery table or a BigQuery result.
+
+
+Properties
+----------
+**referenceName:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**projectID:** The ID of the project in Google Cloud. (Macro-enabled)
+
+**jsonKeyFilePath:** The credential json key file path. (Macro-enabled)
+[How to get the keyfile](https://cloud.google.com/storage/docs/authentication#generating-a-private-key)
+
+**inputTableId:** The BigQuery table to read from, in the form [optional projectId]:[datasetId].[tableId], Example:
+publicdata:samples.shakespeare. Note that if the import query is specified, this table should be a empty table
+with the query result schema. User need to first create such a table.(Macro-enabled)
```

Comment:
This description doesn't make any sense to me. If I want to read from `publicdata:samples.shakespeare` using an import query (as below), why would I need to create an empty table at the same location? Makes no sense. What if I can't create a table there? I probably wouldn't have permission to create a table at "public data".